# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

AlignActionGoal action_goal
AlignActionResult action_result
AlignActionFeedback action_feedback
